docs/guidance/map/connecting_publications.md

```diff
@@ -0,0 +1,429 @@
+
+If publications have overlapping fields and inconsistent mappings, then you ought to use [links](../../schema/reference.md#link) to connect releases across the publications. If, like FTS, a publication's releases are derived from another publication's data, the derivative publication ought to use the 'canonical' [link relation type](../../schema/codelists.md#link-relation-type) to refer to the releases in the authoritative publication. If a publication's releases follow another publication's releases, the subsequent publication ought to use the 'prev' link relation type to refer to the earlier releases.
+
+Building on the PCS and FTS example, the publishers agreed that PCS is the canonical publication. Therefore, when re-publishing releases from PCS, FTS mints a new ocid using its own [ocid prefix](../../schema/identifiers.md#registered-prefixes) and links back to the PCS release using the 'canonical' link relation type:
+
+`````{tab-set}
+````{tab-item}  PCS release
+```json
+{
+  "id": "rls-1-JAN468109",
+  "ocid": "ocds-r6ebe6-0000710562",
+  "date": "2023-01-09T00:00:00Z",
+  "tag": [
+    "planning"
+  ],
+  "parties": [
+    {
+      "id": "org-1",
+      "identifier": {
+        "legalName": "Fife Council"
+      },
+      "name": "Fife Council",
+      "roles": [
+        "buyer"
+      ]
+    }
+  ]
+}
+```
+````
+````{tab-item}  FTS release
+```json
+{
+  "id": "000521-2023",
+  "ocid": "ocds-h6vhtk-0395de",
+  "date": "2023-01-09T12:26:29Z",
+  "tag": [
+    "planning"
+  ],  
+  "parties": [
+    {
+      "id": "GB-FTS-2273",
+      "identifier": {
+        "legalName": "Fife Council"
+      },
+      "name": "Fife Council",
+      "roles": [
+        "buyer"
+      ]
+    }
+  ],
+  "links": [
+    {
+      "href": "https://api.publiccontractsscotland.gov.uk/v1/Notice?id=ocds-r6ebe6-0000710562",
+      "rel": "canonical"
+    }
+  ]
+}
+```
+````
+`````
+
+### How to reuse ocids across publications
+
+There are two scenarios in which reusing ocids across publications might be possible: publications with distinct coverage of OCDS fields and publications that implement consistent OCDS mappings.
+
+### Publications with distinct coverage
+
+If two publications cover disjoint sets of OCDS fields, with the exception of the release-level required fields (`ocid`, `id`, `date` and `tag`), then the publications can reuse ocids so that users can merge releases.
+
+The publishers would need to agree the process and responsibilities for minting ocids. For example: who can mint ocids; if an ocid cannot be calculated deterministically from the input data, then how to lookup whether an ocid for a given process already exists, and how to share the new ocid for others to find. They would also need to implement the agreed approach, whether this means reporting a new ocid to a central registry, or requesting the ocid for a given procedure from a shared "ocid issuer" service, etc.
```

Comment:
It might be simpler to discuss the assignment of new OCIDs as a separate subject. In the PR for `identifiers.md`, it's clarified that there's a delegation process. It might be too much to cover that in this example.

Reply:
Not sure how much we want to cover here (there are already many examples), versus in a solution to https://github.com/open-contracting/standard/issues/364

Reply:
Yes, the second paragraph does feel very information-heavy. I think the information probably fits better with the solution to #364, although that it is of broader scope than simply republishing.

Reply:
I've removed the content in 413c39012e15236ef8bb347b3b41df947e6e4c71 and added it to #364